data/presets.yaml

```diff
@@ -1436,8 +1436,6 @@ en:
       maxheight:
         # maxheight=*
         label: Max Height
-        # maxheight field placeholder
-        placeholder: '4, 4.5, 5, 14''0", 14''6", 15''0"'
```

Comment:
This placeholder could probably be restored, but rewritten so that it only encourages the use of `.` as a decimal separator but doesn’t encourage putting `'` and `"` directly in the field.